workspace_tools/targets.py

```diff
@@ -842,6 +842,20 @@ def __init__(self):
         NRF51822.__init__(self)
         self.extra_labels = ['NORDIC', 'MCU_NRF51822', 'MCU_NORDIC_16K']
         self.macros = ['TARGET_NRF51822']
+		
+class DELTA_DFCM_NNN40(NRF51822):
+    def __init__(self):
+        NRF51822.__init__(self)
+        self.core = "Cortex-M0"
+        self.extra_labels = ['NORDIC', 'MCU_NRF51822', 'MCU_NORDIC_16K']
+        self.macros = ['TARGET_NRF51822']
+
+class DELTA_DFCM_NNN40_OTA(NRF51822):
+    def __init__(self):
+        NRF51822.__init__(self)
+        self.core = "Cortex-M0"
+        self.extra_labels = ['NORDIC', 'MCU_NRF51822', 'MCU_NORDIC_16K', 'DELTA_DFCM_NNN40']
+        self.MERGE_SOFT_DEVICE = False
```

Comment:
Because this target, DELTA_DFCM_NNN40, inherits form NRF51822 that supports ARM and GCC_ARM toolchain, I delete the declaration of self.supported_toolchains.
I think DELTA_DFCM_NNN40 also can support the same toolchain as NRF51822, right?
Is it correct?